### ISSUE TITLE
Removes imports directives

### ISSUE THREAD

**Issue by arnaspet** [incu6us/goimports-reviser#80]
hey, so I noticed that goimports-reviser removes comment directives by using `go/ast.go (g *CommentGroup) Text()` as it ignores directives in this line
```go
if isDirective(c) {
    // Ignore //go:noinline, //line, and so on.
    continue
}
```

I managed to fix this issue locally by copy-pasting `Text` method and changing few lines, but that does not seem like a **clean**
solution.

#### Input
```go
import (
	"log"

	"github.com/incu6us/goimports-reviser/testdata/innderpkg"

	"bytes"
	"go/ast" //nolint:gomnd

	"github.com/pkg/errors"
)
```

#### Expected
```go
import (
	"bytes"
	"go/ast" //nolint:gomnd
	"log"

	"github.com/pkg/errors"

	"github.com/incu6us/goimports-reviser/testdata/innderpkg"
)
```

#### Actual
```go
import (
	"bytes"
	"go/ast" //

	"github.com/pkg/errors"

	"github.com/incu6us/goimports-reviser/testdata/innderpkg"
)
```

**Comment by incu6us** [comment]
I have test for this and it is passed.
https://github.com/incu6us/goimports-reviser/blob/master/reviser/file_test.go#L254

Describe more: version and params to your execution

**Comment by james-d-elliott** [comment]
I can replicate this with version 3.1.1 with the only option being `-file-path` with the following complete example:

```go
package main

import (
	"crypto/sha1" //nolint:gosec // Usage is for collision avoidance not security.
	"fmt"
)

func main() {
	s := sha1.New()

	s.Write([]byte("example"))

	fmt.Printf("%x\n", s.Sum(nil))
}
```

**Comment by incu6us** [comment]
@james-d-elliott thanks, will check.
Btw, -file-path option is deprecated and going be removed soon

**Comment by arnaspet** [comment]
@incu6us your test is with comment, you need to test it with **directive**

**Comment by darwayne** [comment]
+1 was also running into this .. went ahead and did the workaround that @arnaspet suggested
  - was trying to set a `//nolint:` directive on one my imports when I saw this

**Comment by incu6us** [comment]
fixed in v3.2.1
